package.json

```diff
@@ -162,10 +162,10 @@
     "@types/webpack-hot-middleware": "2.25.9",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32369&commitId=a4a2dd110c126e49b66e7e5f9190e7e602a5ba11&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|34| Changed|
|Avatar Converged.badgeMask.chromium.png|20| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2215| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>